Dockerfile.pr-builder

```diff
@@ -1,23 +1,23 @@
-FROM debian:9.5
+FROM ubuntu:18.04
 MAINTAINER Hail Team <hail@broadinstitute.org>
 
 USER root
 
+ENV LANG C.UTF-8
+
+ENV DEBIAN_FRONTEND noninteractive
+
 # - libnlopt-dev is for nloptr, a dependency of SKAT:
 #   https://github.com/jyypma/nloptr/issues/40
 #
 # - r-cran-ncdf4, r-cran-rnetcdf, libcurl4-openssl-dev, and libxml2-dve are
 # - depenendencies for the three biolite packages
 RUN apt-get update && \
-  apt-get -y install gnupg2 lsb-release curl apt-transport-https && \
-  curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - && \
-  echo "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable" >> /etc/apt/sources.list && \
-  echo "deb http://ftp.debian.org/debian $(lsb_release -cs) main contrib non-free" >> /etc/apt/sources.list && \
-  apt-get update && \
   apt-get -y install \
     bash \
     cmake \
-    docker-ce \
+    curl \
+    docker.io \
```

Comment:
Might be worth adding `jq` while you're at it.

Reply:
Actually, this is the wrong docker file.

Reply:
Oh, god, please don't ask me to change the pr-builder image again!  But yes, let's save it for the next update.